### DISCUSSION TITLE
NVMC table to see which report amended another?

### DISCUSSION THREAD

**Discussion by tamihuber** [paulwinterbauer/NBIC#108] [Q&A]
http://serc4.si.edu:8001/NVMC/review/3874047/ - this says I voided this as an amended record, but there are no other NVMC records for this vessel, so I don't understand what's up. Is there a table where the amended report and the one that it was amended to are shown?

**Comment by paulwinterbauer** [comment]
Yes, the table is called noa_metric in the NVMC_ws database.  

One quirk of it is that it uses the id field of the NVMC record rather than the NVMC_ID field.  
So to see all of the noa_metric records you would do a query like so:
`Select nm.* 
From NVMC_ws.noa_metric nm Inner Join NVMC_ws.ws_arrivals wa On nm.noa_id = wa.id
Where wa.NVMC_ID = 3874047;`

That query shows that the record was voided by record id=1729432.  However, that record seems to have been deleted.

@karlsi, any way to see how that record was deleted?

**Reply by karlsi** [reply]
Doesn't look like it
